notesapi/v1/search_indexes.py

```diff
@@ -12,6 +12,7 @@ class NoteIndex(indexes.SearchIndex, indexes.Indexable):
     created = indexes.DateTimeField(model_attr='created')
     updated = indexes.DateTimeField(model_attr='updated')
     tags = indexes.CharField(model_attr='tags')
+    data = indexes.CharField(use_template=True)
```

Comment:
Yes, the index does need to be recreated. I'm not sure about the performance impact. @dan-f any thoughts?